site/src/api/typesGenerated.ts

```diff
@@ -243,6 +245,8 @@ export interface Template {
   readonly active_version_id: string
   readonly workspace_owner_count: number
   readonly description: string
+  readonly max_ttl_ms: number
+  readonly min_autostart_interval_ms: number
```

Comment:
**Question(for FE)**: Could I use these to set min/max and validations on the `WorkspaceScheduleForm` ?

I think I can pass the template down into those components and add that logic. Is there any concerns or gotchas or notes about me doing that @johnstcn ?